.travis.yml

```diff
@@ -38,9 +38,9 @@ script:
 
 deploy:
   provider: npm
-  email: tobias.bieniek@gmx.de
+  email: stefan.penner+ember-cli@gmail.com
   api_key:
-    secure: V/2EC1mCfWuYui+MyzrOnZqCWhwJDTEyhsrxlAPIN0T/BbHQdFgYlu+V4Pi03D/Mn0/0x50pcRmvA9A2SfylollRRTZNlnsHvc9QVT+SAp5ww49umhbOLUlEa9+HvufT1U8+t5kUYzBJ8EJJb0fXpqtpvgdf6Q5ZFeOiQ783lYg=
+    secure: iBMJGoVWQPcqRBVVzKF0Z18BU3JdBi6eu03W7rZxGZ1pmsUCMHQ3D5YzC7bktO1d3cq4EZYLYNvZpcpU5wv7DdZ73/TNDfIysDg41Le/dwB1LD1/Z2aOdwy1hNRPCzoK6ciUysjZc7KEkgUQV5Tj5d/QbneybdUP33PvER1Q39Q=
   on:
     tags: true
```

Comment:
Oh, missed that since it wasn't in green or red. :+1: Thanks for the remedial lesson in reading! 😜